# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/raku-librarymake/ld.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- P6-LibraryMake-7aae514/lib/LibraryMake.pm6.vanilla	2021-07-11 15:24:19.191000000 +0200
+++ P6-LibraryMake-7aae514/lib/LibraryMake.pm6	2021-07-11 15:26:16.309000000 +0200
@@ -184,7 +184,12 @@
     }
 
     for %vars.kv -> $k, $v {
-      %vars{$k} [R//]= %*ENV{$k};
+      if %*ENV<SDECFG_VERBOSE>:exists and $k eq 'LD' {
+          # Inside T2 %LD% should be set to $CC
+          %vars{$k} [R//]= %*ENV<CC>;
+      } else {
+          %vars{$k} [R//]= %*ENV{$k};
+      }
     }
 
     return %vars;
